packages/eslint-config-universe/shared/typescript-analysis.js

```diff
@@ -4,10 +4,31 @@ module.exports = {
     {
       files: ['*.ts', '*.tsx', '*.d.ts'],
       rules: {
+        '@typescript-eslint/await-thenable': 'warn',
         '@typescript-eslint/no-for-in-array': 'error',
         '@typescript-eslint/no-throw-literal': 'warn',
+        '@typescript-eslint/no-confusing-void-expression': 'warn',
+        '@typescript-eslint/no-misused-promises': [
+          'error',
+          {
+            checksVoidReturn: false,
+          },
+        ],
+        '@typescript-eslint/no-floating-promises': 'warn',
         '@typescript-eslint/prefer-nullish-coalescing': 'warn',
         '@typescript-eslint/prefer-optional-chain': 'warn',
+        '@typescript-eslint/no-confusing-non-null-assertion': 'warn',
+        '@typescript-eslint/no-extra-non-null-assertion': 'warn',
+        '@typescript-eslint/prefer-as-const': 'warn',
+        '@typescript-eslint/prefer-includes': 'warn',
```

Comment:
I think it's OK, but confirm that Hermes supports `[].includes`.

Reply:
Looks like it: https://github.com/facebook/hermes/blob/677206674038a650e9da8a1c92e637cdba2eda7b/tools/hvm-bench/stringIncludes.js#L18

Reply:
Though no modules in expo/expo actually use this config.